tests/js/common/aql/aql-arangosearch-inverted-index-search-alias-community.js

```diff
@@ -0,0 +1,974 @@
+/*jshint globalstrict:false, strict:false, maxlen: 500 */
+/*global fail, assertEqual, assertNotEqual, assertTrue */
+
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2022 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Alexey Bakharev
+////////////////////////////////////////////////////////////////////////////////
+
+let arangodb = require("@arangodb");
+let analyzers = require("@arangodb/analyzers");
+let db = arangodb.db;
+let internal = require("internal");
+let jsunity = require("jsunity");
+let errors = arangodb.errors;
+let dbName = "InvertedIndexSearchAliasSuiteCommunity";
+let isEnterprise = internal.isEnterprise();
+const isCluster = require("internal").isCluster();
+const { triggerMetrics } = require("@arangodb/test-helper");
+const { checkIndexMetrics } = require("@arangodb/test-helper-common");
+
+function IResearchInvertedIndexSearchAliasAqlTestSuiteCommunity() {
+    let cleanup = function () {
+        db._useDatabase("_system");
+        try { db._dropDatabase(dbName); } catch (err) { }
+    };
+
+    var assertInvertedIndexCreation = function (collName, inedxDefinition, isNewlyCreated = true) {
+        var collection = db._collection(collName);
+        var res;
+        assertTrue(collection != null);
+        res = collection.ensureIndex(inedxDefinition);
+        assertEqual(res["isNewlyCreated"], isNewlyCreated);
+    };
+
+    return {
+        setUpAll: function () {
+            cleanup();
+
+            db._createDatabase(dbName);
+            db._useDatabase(dbName);
+
+            var analyzers = require("@arangodb/analyzers");
+            analyzers.save("AqlAnalyzerRound", "aql", { queryString: "RETURN ROUND(@param)" }, ["frequency", "norm", "position"]);
+            analyzers.save("myDelimiterAnalyzer", "delimiter", { delimiter: " " }, ["frequency", "norm", "position"]);
+            analyzers.save("AqlAnalyzerHash", "aql", { queryString: "return to_hex(to_string(@param))" }, []);
+            analyzers.save("norm_lower", "norm", { locale: "de.utf-8", case: "lower" }, ["frequency", "norm", "position"]);
+            analyzers.save("norm_lower_dup", "norm", { locale: "de.utf-8", case: "lower" }, ["frequency", "norm", "position"]);
```

Comment:
Encoding does not need to be set anymore since 3.9
```suggestion
            analyzers.save("norm_lower", "norm", { locale: "de", case: "lower" }, ["frequency", "norm", "position"]);
            analyzers.save("norm_lower_dup", "norm", { locale: "de", case: "lower" }, ["frequency", "norm", "position"]);
```

Reply:
Ok, but I think we can leave this as is while it is not deprecated.